blocks/pure_cookies_notice/controller.php

```diff
@@ -198,36 +204,41 @@ private function shouldShowAgreement()
         return $this->shouldShowAgreementResult;
     }
 
-    private function generateStylesheet()
+    private static function generateCss(Controller $controller)
     {
         $lines = ['<style>'];
-        $lines[] = "#pure-cookies-notice-{$this->bID} {";
-        if (!empty($this->textColor)) {
-            $lines[] = "\tcolor: {$this->textColor};";
+        $lines[] = "#pure-cookies-notice-{$controller->bID} {";
+        if (!empty($controller->textColor)) {
+            $lines[] = "\tcolor: {$controller->textColor};";
         }
-        if (!empty($this->backgroundColor)) {
-            $lines[] = "\tbackground: {$this->backgroundColor};";
+        if (!empty($controller->backgroundColor)) {
+            $lines[] = "\tbackground: {$controller->backgroundColor};";
         }
         $lines[] = '}';
-        $lines[] = "#pure-cookies-notice-{$this->bID} a {";
-        if (!empty($this->linkColor)) {
-            $lines[] = "\tcolor: {$this->linkColor};";
+        $lines[] = "#pure-cookies-notice-{$controller->bID} a {";
+        if (!empty($controller->linkColor)) {
+            $lines[] = "\tcolor: {$controller->linkColor};";
         }
-        elseif (!empty($this->textColor)) {
-            $lines[] = "\tcolor: {$this->textColor};";
+        elseif (!empty($controller->textColor)) {
+            $lines[] = "\tcolor: {$controller->textColor};";
         }
         $lines[] = '}';
-        $lines[] = "#pure-cookies-notice-{$this->bID} .pure-cookies-notice-close-button {";
-        if (!empty($this->textColor)) {
-            $lines[] = "\tcolor: {$this->textColor};";
-            $lines[] = "\tborder-color: {$this->textColor};";
+        $lines[] = "#pure-cookies-notice-{$controller->bID} .pure-cookies-notice-close-button {";
+        if (!empty($controller->textColor)) {
+            $lines[] = "\tcolor: {$controller->textColor};";
+            $lines[] = "\tborder-color: {$controller->textColor};";
         }
         $lines[] = '}';
         $lines[] = '</style>';
 
         return implode("\n", $lines);
```

Comment:
maybe need use PHP_EOL?

Reply:
Well, it's the same, both will work.
BTW all the other code uses the Unix line ending (`\n`), so I'd prefer `\n`